iceoryx_examples/icediscovery/README.md

```diff
@@ -0,0 +1,13 @@
+# icediscovery
```

Comment:
These changes will conflicting with the changes from #1105

Reply:
Yes, but can be resolved easily (CMake and readme). #1105 will be merged before this.